kikuchipy/signals/ebsd_master_pattern.py

```diff
@@ -270,6 +277,117 @@ def set_phase_parameters(
         phase = {k: v for k, v in inputs.items() if v is not None}
         _update_phase_info(self.metadata, phase, number)
 
+    def get_patterns(
```

Comment:
I think the docstring needs more information:
* Only a Lambert projection is allowed (this could be stated in the top description)
* If the master pattern phase has a non-centrosymmetric point group, both the northern and southern hemispheres must be provided (I think it would be okay to add this in a Notes section after Returns)
* The sample reference frame must be stated unambiguously here (I think top description), preferably also with a reference to the reference frame user guide (Notes).
* Shape of the output signal (in Returns).